src/core/vertex-ai-connector.ts

```
@@ -129,36 +129,64 @@
       }
 
-      // Initialize authentication
+      if (!this.config.location) {
+        throw new Error("Location is required for Vertex AI initialization");
+      }
+
+      // Initialize authentication with comprehensive credential handling
       this.auth = new googleAuthModule.GoogleAuth({
         projectId: this.config.projectId,
         keyFilename: this.config.serviceAccountPath,
         credentials: this.config.credentials,
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)